### ISSUE TITLE
SchemaName parameter issue

### ISSUE THREAD

**Issue by jmazier** [b00ted/serilog-sinks-postgresql#16]
Hi,

First of all, thanks for the amazing job done on this Serilog sink, it has been really useful so far.

But, I'm facing an issue when trying to **provide a different SchemaName than the default one** (public) to PostgresSQL method : 
` .PostgreSQL(connectionstring, tableName, columnWriters, useCopy: true, needAutoCreateTable: true, schemaName: "testLogs")`

It seems like the SchemaName is ignore or never use. Do you have any more information about this concern ? Have you ever use this parameter ?

Thanks a lot !

**Comment by dhabierre** [comment]
Hi @jmazier,

I think you are right, schema name seems not to be used when creation the table:

```cs
private static string GetCreateTableQuery(string tableName, IDictionary<string, ColumnWriterBase> columnsInfo)
{
    var builder = new StringBuilder("CREATE TABLE IF NOT EXISTS ");
    builder.Append(tableName);
    ...
}
```

It should be something like this:

```cs
private static string GetCreateTableQuery(string schemaName, string tableName, IDictionary<string, ColumnWriterBase> columnsInfo)
{
    var builder = new StringBuilder("CREATE TABLE IF NOT EXISTS ");
    builder.Append($"{GetSchemaPrefix(schemaName)}{tableName}");
    ...
}

private static string GetSchemaPrefix(string schemaName)
  => !string.IsNullOrEmpty(schemaName) ? $"{schemaName}." : string.Empty;
```

**Comment by dhabierre** [comment]
[Pull Request](https://github.com/b00ted/serilog-sinks-postgresql/pull/19)

**Comment by b00ted** [comment]
Fixed in version 2.0.1
